lib/flatpak-installation.c

```diff
@@ -1976,6 +1977,41 @@ flatpak_installation_uninstall (FlatpakInstallation    *self,
                                 gpointer                progress_data,
                                 GCancellable           *cancellable,
                                 GError                **error)
+{
+  return flatpak_installation_uninstall_full (self, FLATPAK_UNINSTALL_FLAGS_NONE,
+                                              kind, name, arch, branch,
+                                              progress, progress_data,
+                                              cancellable, error);
+}
+
+/**
+ * flatpak_installation_uninstall_full:
+ * @self: a #FlatpakInstallation
+ * @flags: set of #FlatpakUninstallFlags flag
+ * @kind: what this ref contains (an #FlatpakRefKind)
+ * @name: name of the app or runtime to uninstall
+ * @arch: architecture of the app or runtime to uninstall
+ * @branch: name of the branch of the app or runtime to uninstall
+ * @progress: (scope call) (nullable): the callback
+ * @progress_data: (closure progress) (nullable): user data passed to @progress
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Uninstall an application or runtime.
+ *
+ * Returns: %TRUE on success
```

Comment:
Nitpick: Needs a `Since: xx` line giving the version number of the first release which would contain this new API.

Reply:
Sure